test/util/mock_canvas.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:ui';
+
+import 'package:test/fake.dart';
+
+class MockCanvas extends Fake implements Canvas {
```

Comment:
I wonder if we could start using a mock (like mockito) library instead of doing that manually, don't need to change for this PR, but something to think about on the next PRs

Reply:
Yeah that could be a good idea!